tests/build_test.sh

```diff
@@ -877,4 +898,20 @@ function test_kernel_build_menu_inside_env()
   assertEquals "($LINENO)" "$expected_result" "$output"
 }
 
+function test_kernel_build_clean_inside_env()
+{
+  local expected_result
+  local output
+  local env_output="${KW_CACHE_DIR}/fake_env"
+  build_config=()
+
+  mk_fake_kw_env
+
+  parse_configuration "$SAMPLES_DIR/build_no_log.config" build_config
```

Comment:
Please, refer to: https://kworkflow.org/content/codingstyle.html#string-concatenation